docs/graphql-concepts.md

```diff
@@ -0,0 +1,218 @@
+## The basics
+
+Mailchimp Open Commerce is an API-first, modular commerce stack. The API implements all features in GraphQL. MOC GraphQL is accessible at `/graphql` on the API server port.
+To use GraphQL, just POST to `/graphql`.
+
+The GraphQL server is implemented using [Apollo Server](https://www.apollographql.com/docs/apollo-server/). It is compatible with [Apollo Client](https://www.apollographql.com/docs/react/) or [Relay Modern](https://facebook.github.io/relay/) for client development.
+
+As you follow the guides for GraphQL, you'll find the concepts in this guide very valuable.
+
+## GraphQL Clients
+
+We recommend using a standalone GraphQL client to connect and make requests. Here are a few popular ones:
+- [GraphQL Playground](https://github.com/prismagraphql/graphql-playground)
+- [Altair GraphQL Client](https://altair.sirmuel.design/)
+- [GraphiQL Standalone App](https://github.com/skevy/graphiql-app)
```

Comment:
Linking to the readme https://github.com/skevy/graphiql-app#readme
Could save a couple a scrolls